include/configs/hmi.h

```diff
@@ -124,6 +124,7 @@
 	"error_cnt=0\0"								\
 	"error_max=3\0"								\
 	"err=setexpr error_cnt $error_cnt + 1; run save; reset;\0"		\
+	"clean=setenv tmp; setenv filesize;\0"					\
```

Comment:
Enlighten me, because I don't see where we use the `tmp` and `filesize` env-var, so why are we clearing them?

Reply:
And thou shall be enlightened, for they will not establish in the explicit realm of U-Boot. In contrast! The first is formed by the casting of the U-Boot scroll, whereas the latter is a spawn from the kernel being loaded.

They are relics made permanent after the fall of what we call the green universe. 

tl;dr;
After blue/green reload these variables are saved in the environment. tmp is created by the script and filesize due to the script loading the kernel.